src/ManifoldDiff.jl

```diff
@@ -203,7 +203,7 @@ function __init__()
     # There is likely no way to set defaults without Requires.jl
     @require FiniteDifferences = "26cc04aa-876d-5657-8c51-4c34ba976000" begin
         if default_differential_backend() === NoneDiffBackend()
-            set_default_differential_backend!(FiniteDifferencesBackend())
+            set_default_differential_backend!(FiniteDifferencesBackend()) #This expects a method in the inner ()?
```

Comment:
When trying to render the docs, this line reports

```
┌ Warning: Error requiring `FiniteDifferences` from `ManifoldDiff`
│   exception =
│    MethodError: no method matching ManifoldDiff.FiniteDifferencesBackend()
│    
│    Closest candidates are:
│      ManifoldDiff.FiniteDifferencesBackend(::TM) where TM
│       @ ManifoldDiff ~/Repositories/Julia/ManifoldDiff.jl/src/finite_differences.jl:7
```

which for me means
1. we do not yet use extensions here?
2. This line seems to be left out in some update recently?

Reply:
I think this may be the same as #26 . So far the only options are to either have it like it is now or remove the `@require` block and not have the default.

Reply:
But did that then ever work? I mean this was some documentation, that at some point did render?

And why am I running into this with Julia 1.9 Should that not use extensions then? Then we could at least fix this on current Julia versions that way.

Reply:
It's just a warning and IIRC documentation doesn't need that default. So it is rendered just fine from master. And, as far as I know, there is no way to "fix" this, only remove the functionality.

Reply:
Well it is a warning, but since that does not load this one require block we also have one undocumented string – in Manopt I went to strict mode where that would error having doc-strings that are not part of the documentation.

Reply:
You can load `ManifoldDiff` first in `make.jl` here and the warning will disappear.